src/plots/geo/zoom.js

```diff
@@ -164,6 +165,7 @@ function zoomNonClipped(geo, projection) {
 
         didZoom = true;
         geo.render();
+        geo.graphDiv.emit('plotly_relayouting', {'projection.scale': projection.scale() / geo.fitScale});
```

Comment:
I think you're missing one `.emit` call in `zoomClipped`

https://github.com/plotly/plotly.js/blob/7a66dafc5e2b38a907b7757849240f432d6e2372/src/plots/geo/zoom.js#L195-L287

Reply:
Thanks for pointing that out @etpinard. I also just realized that I should output more information on `plotly_relayouting` to be on par with `plotly_relayout`. Right now, we're missing latitude and longitude.

Reply:
In 2545746698588ac3f3edaa91bec2a1ac98ac33b3, I check that each key in the event data of `plotly_relayout` is also present in the event data of `plotly_relayouting`.

Reply:
Fantastic!